README.md

```diff
@@ -26,20 +26,20 @@ Generator supports Node.js and [all browsers] from IE 11.
 
 ## Security
 
-*Good article about random generators theory:
+*See a good article about random generators theory:
 [Secure random values (in Node.js)]*
 
 ### Unpredictability
 
-Nano ID doesn’t use unsafe `Math.random()`. It use `crypto` module in Node.js
+Instead of unsafe `Math.random()` Nano ID uses `crypto` module in Node.js 
 and Web Crypto API in browsers.
 
 ### Uniformity
 
 `random % alphabet` is a popular mistake in ID generator. Change to get some
 symbols will be lower and it will reduce amount of guesses in bruteforcing.
```

Comment:
I've failed to parse `Change to get some symbols will be lower and it will reduce amount of guesses in bruteforcing`. It's worth to rewrite the sentence completely.